docusaurus.config.js

```diff
@@ -151,8 +151,8 @@ module.exports = {
     announcementBar: {
       id: "summit",
       content: renderAnnouncementBar(
-        "Submit your stories to Pulsar Summit EMEA! 📣",
-        "https://sessionize.com/pulsar-virtual-summit-europe-2024"
+        "Join Pulsar Virtual Summit Europe 2024 🗓️",
```

Comment:
Asking to get the free pass and also adding the date?

```suggestion
        "Get your free pass for Pulsar Virtual Summit Europe 2024 on May 14, 2024 🗓️",
```

Reply:
It looks like it fits the mobile screen.

<img width="533" alt="Screenshot 2024-03-14 at 11 57 11 AM" src="https://github.com/apache/pulsar-site/assets/9302460/f858d3aa-669a-48a0-b8a8-51090056445f">